app.py

```diff
@@ -1,13 +1,127 @@
-from flask import Flask
-import requests
+from flask import Flask, request
+import requests, json, pymongo
+from pymongo import MongoClient
 
+def connect():
+    connection = MongoClient("ds031257.mlab.com",31257)
+    handle = connection["general_info_database"]
+    handle.authenticate("admin","admin1")
+    return handle
+
+handle=connect()
+db = handle.general_info_database
 app = Flask(__name__)
+URL = 'doc.json'
+db_users = handle.users_database
+
+class Course:
```

Comment:
If you want an ODM layer to automatically translate Python `class`es into Mongo documents, I recommend [mongoengine](https://mongoengine-odm.readthedocs.io/tutorial.html). It'll be a lot easier than rolling your own custom thing.